conf.yaml

```diff
@@ -46,6 +46,8 @@ repos:
     swiftype:             https://github.com/elastic/swiftype-doc-placeholder.git
     terraform-provider-ec: https://github.com/elastic/terraform-provider-ec.git
     enterprise-search-pubs:  https://github.com/elastic/enterprise-search-pubs.git
+    enterprise-search-python: https://github.com/elastic/enterprise-search-python
+    enterprise-search-ruby: https://github.com/elastic/enterprise-search-ruby
```

Comment:
I think it would be better if these were all alphabetized (I guess I didn't catch it when `enterprise-search-pubs` was added).

Reply:
:+1: fixed